test/unit/person_test.rb

```diff
@@ -17,6 +17,25 @@ class PersonTest < ActiveSupport::TestCase
                   }, person.search_index)
   end
 
+  test "truncates the biography sent to rummager if not currently holding a role" do
```

Comment:
Minor quibble - feel free to ignore.
I think the test is for both cases - i.e. the assertions are checking both a person not in a role and a person in a role's biography in the search index. Perhaps a test name like "it sends the appropriate biography to rummager" or somesuch? might help make it more explicit for when someone's scanning the tests quickly.